### PR TITLE
Update branch

### PR DESCRIPTION
**Pull request by hRobert12**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158698937%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158715616%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158811138%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158811647%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158811766%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158812103%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158839474%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158868276%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23discussion_r45570373%22%2C%20%22https%3A//github.com/hRobert12/My-games/pull/9%23discussion_r45576748%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hRobert12/My-games/pull/9%23issuecomment-158698937%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ca%20href%3D%5C%22https%3A//app.review.ninja/hRobert12/My-games/pull/9%5C%22%20target%3D%5C%22_blank%5C%22%3E%3Cimg%20src%3D%5C%22https%3A//app.review.ninja/46381900/pull/9/badge%5C%22%20alt%3D%5C%22ReviewNinja%5C%22/%3E%22%2C%20%22created_at%22%3A%20%222015-11-22T01%3A34%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/8714728%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/review-ninja%22%7D%7D%2C%20%7B%22body%22%3A%20%22%5E%5E%20ummmmm...%22%2C%20%22created_at%22%3A%20%222015-11-22T05%3A55%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22well%20that%27s%20a%20disaster%22%2C%20%22created_at%22%3A%20%222015-11-22T22%3A31%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-11-22T22%3A33%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%22%2C%20%22created_at%22%3A%20%222015-11-22T22%3A33%3A22Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22...%22%2C%20%22created_at%22%3A%20%222015-11-22T22%3A34%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-11-23T03%3A04%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-11-23T07%3A27%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20d2d2f6c86b42f5348391c457095c002293887444%20random_game.py%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/hRobert12/My-games/pull/9%23discussion_r45570373%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22this%20used%20to%20go%20to%20home%20screen%22%2C%20%22created_at%22%3A%20%222015-11-23T03%3A06%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%2C%20%7B%22body%22%3A%20%22It%20will%20soon%20%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-11-23T07%3A26%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/hRobert12%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20random_game.py%3AL16-22%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/hRobert12%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15806419%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/hRobert12'></a>
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/hRobert12/My-games/pull/9#issuecomment-158698937'>General Comment</a></b>
- <a href='https://github.com/review-ninja'></a> <a href="https://app.review.ninja/hRobert12/My-games/pull/9" target="_blank">ReviewNinja
- <a href='https://github.com/hRobert12'></a> ^^ ummmmm...
- <a href='https://github.com/hRobert12'></a> well that's a disaster
- <a href='https://github.com/hRobert12'></a> :+1:
- <a href='https://github.com/hRobert12'></a> :-1:
- <a href='https://github.com/hRobert12'></a> ...
- [x] <a href='#crh-comment-Pull d2d2f6c86b42f5348391c457095c002293887444 random_game.py 4'></a> &nbsp;<b><a href='https://github.com/hRobert12/My-games/pull/9#discussion_r45570373'>File: random_game.py:L16-22</a></b>
- <a href='https://github.com/hRobert12'></a> this used to go to home screen
- <a href='https://github.com/hRobert12'></a> It will soon :+1:

<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/9?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/9?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/hRobert12/My-games/pull/9?approve=0'></a>&nbsp;<a href='https://github.com/hRobert12/My-games/pull/9'></a>
<a href='#crh-end'></a>

[Review on Reviewable](https://reviewable.io/reviews/hrobert12/my-games/9)

---

---

---

---

---

---

---

---

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/hRobert12/My-games/pull/9" target="_blank">ReviewNinja

**Comment by hRobert12** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/hRobert12/My-games/pull/9'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by hRobert12** [comment]
^^ ummmmm...

**Comment by hRobert12** [comment]
well that's a disaster

**Comment by hRobert12** [comment]
:+1:

**Comment by hRobert12** [comment]
:-1:

**Comment by hRobert12** [comment]
...

**Comment by hRobert12** [comment]
:+1:

File: random_game.py [hunk 1]
```diff
@@ -16,12 +16,6 @@
 get_inp = True
 ntg = 0
 
-def goToHomescreen():
```
Comment (hRobert12) [inline random_game.py:19]: this used to go to home screen

Comment (hRobert12) [inline random_game.py:19]: It will soon :+1:

**Comment by hRobert12** [comment]
:shipit:

**Pull request merged**